webapp/locales/de.json

```
@@ -258,7 +258,8 @@
     "newPost": "Erstelle einen neuen Beitrag",
     "success": "Gespeichert!",
     "teaserImage": {
-      "cropperConfirm": "Bestätigen"
+      "cropperConfirm": "Bestätigen",
+      "supportedFormats": "Fügen Sie ein Bild im Dateiformat JPG, PNG oder GIF ein"
```

```suggestion
      "supportedFormats": "Füge ein Bild im Dateiformat JPG, PNG oder GIF ein"
```

This is not so formal … 😎